tests/test_array.py

```diff
@@ -0,0 +1,90 @@
+import av
+import numpy as np
+import pytest
+
+import pixeltable as pxt
+from pixeltable.functions.array import to_audio
+
+from .utils import IN_CI, rerun, skip_test_if_not_installed
+
+
+class TestArray:
+    @pytest.mark.parametrize(
+        'format,downsample', [('wav', False), ('mp3', False), ('mp3', True), ('flac', False), ('mp4', False)]
+    )
+    def test_to_audio(self, format: str, downsample: bool, reset_db: None) -> None:
+        # Load a sample mp3 file to an array
+        audio_data, sample_rate = self._load_sample_audio('./docs/resources/10-minute tour of Pixeltable.mp3')
+        assert audio_data.dtype == np.float32
+        assert audio_data.ndim == 2
+        assert audio_data.shape[0] == 1  # That sample is in mono
+
+        # Use to_audio to encode it to an audio file
+        t = pxt.create_table('test_mp3_to_array_and_back', {'audio_array': pxt.Array[pxt.Float]})  # type: ignore[misc]
+        output_sample_rate = sample_rate // 2 if downsample else sample_rate
+        t.add_computed_column(
+            audio_file=to_audio(
+                t.audio_array, input_sample_rate=sample_rate, format=format, output_sample_rate=output_sample_rate
+            )
+        )
+        update_status = t.insert(audio_array=audio_data)
+        assert update_status.num_rows == 1
```

Comment:
You can use the `validate_update_status` utility instead of coding these assertions separately.